.travis.yml

```diff
@@ -22,25 +26,31 @@ matrix:
     fast_finish: true
     include:
         - php: 5.3
-          env: SYMFONY_DEPRECATIONS_HELPER=weak
+          dist: precise
         - php: 5.3
-          env: COMPOSER_FLAGS="--prefer-lowest" SYMFONY_DEPRECATIONS_HELPER=weak
+          dist: precise
+          env: SYMFONY_DEPRECATIONS_HELPER COMPOSER_FLAGS="--prefer-lowest"
         - php: 5.4
-          env: SYMFONY_DEPRECATIONS_HELPER=weak
         - php: 7.0
-          env: SYMFONY_VERSION=2.7.* SYMFONY_DEPRECATIONS_HELPER=weak
+          env: SYMFONY_DEPRECATIONS_HELPER=weak_vendors SYMFONY_VERSION=2.7.*
```

Comment:
Since it's the same as the default, `SYMFONY_DEPRECATIONS_HELPER` can be removed here.

Reply:
It doesn't work like that sadly

Reply:
I mean it's overwritten, not merged.